maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/index.ts

```diff
@@ -382,38 +395,57 @@ export class Project {
 
 			}
 
-			return autoPresets;
+			return autopresets;
 		}, []);
-	}
 
-	_loadDefaultPreset(): string {
-		let presetFilePath;
-
-		const autoPresets = this._findAutopresets();
-
-		if (autoPresets.length > 1) {
+		if (autopresets.length > 1) {
 			throw new Error(
-				'Multiple autopreset dependencies found in project (' +
-					autoPresets +
-					'): please remove the invalid ones or ' +
+				'Multiple autopreset dependencies found in project ' +
+					`(${autopresets}): please remove the invalid ones or ` +
 					'explicitly define the preset to be used in the ' +
 					'.npmbundlerrc file'
```

Comment:
You didn't add template literals across the whole string because of the line breaking, right?

Reply:
Yes. We could use them, but then we would need to move new lines to the left to remove the indent and it Looks Horrible :tm: